scripts/upgrade-runtime-unsafe.sh

```diff
@@ -153,8 +141,9 @@ fi
 echo "🛂 awaiting runtime upgrade authorization..."
 
 head=$(get_finalized_head)
+head_after=$((head + after))
```

Comment:
calc new height unnecessary when using scheduler.scheduleAfter #615

Reply:
so we dont want to wait in the loop to enact?

```
while [[ $head -ne $head_after ]]; do
  sleep 12s
  head=$(get_finalized_head)
done

echo "⚙️ enacting runtime upgrade... $dryrun"
```

Reply:
I have reverted my changes because it seems to already been PRd
https://github.com/t3rn/t3rn/pull/619